apps/docs/pages/guides/getting-started/tutorials/with-kotlin.mdx

```diff
@@ -0,0 +1,816 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  title: 'Build a Product Management App with Jetpack Compose in Android',
+  description: 'Learn how to use Supabase in your Android Kotlin App.',
+}
+
+<QuickstartIntro />
+
+![manage-product-cover](https://github.com/hieuwu/product-sample-supabase-kt/assets/43868345/1bed0c84-208a-4266-a2ec-2bac50ddf80c)
+
+<Admonition type="note">
+  If you get stuck while working through this guide, refer to the [full example on
+  GitHub](https://github.com/hieuwu/product-sample-supabase-kt).
```

Comment:
copy this to https://github.com/supabase/supabase/tree/master/examples for consistent